apps/frontend/src/pages/projects/projects.tsx

```diff
@@ -50,7 +50,7 @@ const Projects = (): JSX.Element => {
 		[dispatch],
 	);
 
-	const hasProject = projects.length === EMPTY_LENGTH;
+	const hasProject = projects.length !== EMPTY_LENGTH;
```

Comment:
How about using this?
```suggestion
	const hasProject = Boolean(projects.length)
```

Reply:
Good idea

Reply:
But eslint doesn`t allow it

![image](https://github.com/user-attachments/assets/6648cfd6-37e3-41bf-ac27-142297022b63)

Reply:
okay, got it